package.json

```diff
@@ -4,7 +4,7 @@
   "linter-package": true,
   "version": "0.2.1",
   "description": "Linter plugin for Typescript, using tslint",
-  "activationEvents": [],
+  "activationCommands": [],
```

Comment:
But activationEvents has been empty since at least 0.2.0. Prior to that there was an event called linter-tslint:toggle.
If this were restored with the new name I suppose it would just be this:

"activationCommands": {
    "atom-text-editor": ["linter-tslint:toggle"]
},

But why was it removed in the first place?